test/integration/response-test.coffee

```diff
@@ -0,0 +1,341 @@
+{assert} = require('chai')
+
+{runDreddWithServer, createServer} = require('./helpers')
+Dredd = require('../../src/dredd')
+
+
+[
+    name: 'API Blueprint'
+    path: './test/fixtures/response/empty-body-empty-schema.apib'
+  ,
+    name: 'Swagger'
+    path: './test/fixtures/response/empty-body-empty-schema.yaml'
+].forEach((apiDescription) ->
+  describe("Specifying neither response body nor schema in the #{apiDescription.name}", ->
+    describe('when the server returns non-empty responses', ->
+      runtimeInfo = undefined
+
+      beforeEach((done) ->
```

Comment:
I think that server creation for this and subsequent `describe`'s can be moved to upper scope and reused for all assertions, correct? Small things like this add up and are slowly degradeting performance of our test suite.

Reply:
I'm definitely for using `before` where it makes sense, but I wouldn't share `before` blocks between multiple `describe`s, as I think that's over the line of keeping the tests isolated. You don't want to go that way, it easily erodes.